bluez installs stuff into /etc/alsa, which is wrong since the
correct location (also stated in alsa-lib configure.in) is
/usr/share/alsa instead
Upstream report: 
http://permalink.gmane.org/gmane.linux.bluez.kernel/4739
--- Makefile.am~	2010-02-12 20:26:48.000000000 +0100
+++ Makefile.am	2010-03-05 10:17:15.000000000 +0100
@@ -241,7 +241,7 @@
 audio_libasound_module_ctl_bluetooth_la_CFLAGS = @ALSA_CFLAGS@
 
 if CONFIGFILES
-alsaconfdir = $(sysconfdir)/alsa
+alsaconfdir = $(datadir)/alsa
 
 alsaconf_DATA = audio/bluetooth.conf
 endif
